libandroid-navigation-ui/src/main/java/com/mapbox/services/android/navigation/ui/v5/instruction/InstructionView.java

```diff
@@ -316,6 +316,7 @@ public void hideInstructionList() {
    */
   public void showInstructionList() {
     onInstructionListVisibilityChanged(true);
+    instructionLayout.requestFocus();
```

Comment:
Does the `MapView` regain focus okay after the list is hidden?

Reply:
Yup it does. I originally added `abandonFocus` to `hideInstructionList`, I can re-add it if you want

Reply:
No problem, as long as you didn't see any issues